markdown/en/api/ff-searchbox.api.md

```diff
@@ -41,12 +41,12 @@ ___
 | **add-params**&nbsp;(String) (default: empty)| _The parameter string has to be URL-encoded._ With this property you can deliver standard parameters which will then be attached to the search request. Example: add-params="param1=abcd,param2=xyz"|
 | **add-tracking-params**&nbsp;(String) (default: empty)| With this property you can deliver standard parameters which are attached to every tracking request. Example: add-tracking-params="param1=abcd,param2=xyz"|
 | **keep-filters**&nbsp;(Boolean) **Options**: &nbsp;true, &nbsp;false (default: false)| With this property you can determine, if filters which where set before the search (e.g. via ASN) should be kept or discarded.|
-| **keep-url-params**&nbsp;(String) (default: empty)| Determines if parameters which are written into the URL should be kept.|
+| **keep-url-params**&nbsp;(String) (default: empty)| Comma separated values; specifies the url-parameters that should be included in requests to FACT-Finder. If this is set to `all`, every parameter will be used |
```

Comment:
Would it be helpful to explicitly mention FACT-Finder unrelated paramters here? This is a major use case of this attribute, isn't it?

Reply:
It pertains to ALL attributes, non-FF and FF alike. But we could mention it. I'm not sure. What do you think?

Reply:
@toarm what do you think?

Reply:
I'm imagining users who have a setup that requires their own parameters. And they have a question in their mind like _How can I keep my own, FACT-Finder unrelated parameters?_
I personally appreciate buzzwords in documentation. Makes it easier for me to scan. Even `Ctrl+F` might yield some results.
In general, documentation doesn't have to be normalized. Repetition and explicitness are good. So it becomes a piece of empathetic customer care instead of _Amt_ where you only get answers for precisely what you asked.

_A common dialogue I have:_
Me: "I don't eat animals."
Other person: "Oh really? What about fish?"
Me: "..... _(aren't fish animals?)_"

So I'm always a bit careful with _ALL_. It's seems just too much or too implicit to some people.

Reply:
Very well. I will add this use case to the docs.

*Leave it to a vegetarian to bring that up* :wink: